scm/github.go

```diff
@@ -389,35 +389,44 @@ func (s *GithubSCM) CreateTeam(ctx context.Context, opt *TeamOptions) (*Team, er
 			Message: fmt.Sprintf("%+v", opt),
 		}
 	}
-	t, _, err := s.client.Teams.CreateTeam(ctx, opt.Organization.Path, github.NewTeam{
-		Name: opt.TeamName,
-	})
+
+	// first check whether the team with this name already exists on this organization
+	team, _, err := s.client.Teams.GetTeamBySlug(ctx, opt.Organization.Path, slug.Make(opt.TeamName))
 	if err != nil {
-		if opt.TeamName != TeachersTeam && opt.TeamName != StudentsTeam {
-			return nil, ErrFailedSCM{
-				Method:   "CreateTeam",
-				Message:  fmt.Sprintf("failed to create GitHub team %s, make sure it does not already exist", opt.TeamName),
-				GitError: fmt.Errorf("failed to create GitHub team %s: %w", opt.TeamName, err),
+		s.logger.Infof("Team %s not found as expected: %s", opt.TeamName, err)
```

Comment:
I think we should avoid `Info` and use only `Debug` everywhere. Rational: we will not ever show only one of Debug or Info levels.

Reply:
I mean, it is ok to use `Error` level and `Debug` level.

Reply:
Fixed.